governance/governance.md

```diff
@@ -0,0 +1,118 @@
+
+### Helm Org Maintainers
+
+The Helm Org maintainers are responsible for:
+
+* Maintaining the mission, vision, values, and scope of the project
+* Refining the governance and charter as needed
+* Making project level decisions
+* Resolving escalated project decisions when the subteam responsible is blocked
+* Managing the Helm brand
+* Controlling access to Helm assets such as source repositories, hosting, project calendars
+* Handling code of conduct violations
+* Deciding what sub-groups are part of the Helm project
+* Overseeing the resolution and disclosure of security issues
+* Managing financial decisions related to the project
+
+Changes to org maintainers use the following:
+
+* There will be between 3 and 9 people.
+* Any project maintainer is eligible for a position as an org maintainer
+* No one company or organization can employ a simple majority of the org maintainers
+* An org maintainer may step down by emailing the org maintainers mailing list. Within 7 calendar days the [helm mailing list](https://lists.cncf.io/g/cncf-helm) needs to be notified of the change
+* Org maintainers MUST remain active on the project. If they are unresponsive for > 3 months they will lose org maintainership unless a [super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of the other org maintainers agrees to extend the period to be greater than 3 months
+* When there is an opening for a new org maintainer, any person who has made a contribution to any repo under the Helm GitHub org may nominate a suitable project maintainer as a replacement
+  * The nomination period will be three weeks starting the day after an org maintainer opening becomes available
+  * The nomination must be made via the [public Helm mailing list](https://lists.cncf.io/g/cncf-helm/)
+* When nominated individual(s) agrees to be a candidate for maintainership, the project maintainers may vote
+  * The voting period will be open for a minimum of three business days and will remain open until a super-majority of project maintainers has voted
+  * Only current project maintainers are eligible to vote
+  * When the number of nominated individuals matches the number of openings each individual needs to have a yes vote from a super-majority of those that voted
+  * When there are more individuals than open positions voting will use [Condorcet](https://en.wikipedia.org/wiki/Condorcet_method) ranking on [CIVS](http://civs.cs.cornell.edu/) using the [Schulze method](https://en.wikipedia.org/wiki/Schulze_method)
+* When an org maintainer steps down, they become an emeritus maintainer
+
+Once an org maintainer is elected, they remain a maintainer until stepping down (or, in rare cases, are removed). Voting for new maintainers occurs when necessary to fill vacancies. Any existing project maintainer is eligible to become an org maintainer.
+
+The Org Maintainers will select a chair to set agendas and call meetings of the Org Maintainers.
```

Comment:
Something to maybe clarify is if org maintainers can also call meetings. But that feels somewhat implied

Reply:
This is saying that the chair is the one to call meetings of the org maintainers. So, if a non-chair org maintainer thinks the org maintainers need to meet it's on the chair to figure out the time, call the meeting, and handle the agenda for it.

This section was modeled after the CNCF TOC.